### DISCUSSION TITLE
How to use cloud / remote database?

### DISCUSSION THREAD

**Discussion by alexandreelise** [joomlatools/joomlatools-pages-server#96] [Q&A] [answered]
Hi @johanjanssens,

I am trying out joomlatools server today. I was wondering if it was possible to use it with a cloud database.

**Comment by johanjanssens** [comment, marked as answer]
You can easily do, just install a Joomla site and change the Joomla configuration by editing the `configuration.php` to use a cloud database. This works in the exact same way as you would when setting up a cloud / remote database on any other hosting.

**Reply by alexandreelise** [reply]
As I would expect. Thanks for clarifying.
